mcts.py

```diff
@@ -35,6 +36,9 @@
 def D_NOISE_ALPHA(): return 0.03 * 19 / go.N
 
 
+# Progressive widening
+PWIDEN = float(os.environ.get('MINIGO_PWIDEN', 0.0))
```

Comment:
How about a constant instead. I'd like to keep the proliferation of env vars to a minimum. 

I can imagine passing in a config file of ML constants if we want to configure things like this.

Reply:
@amj what are the plans for these sort of tunable things? Stick with env vars? Command line options? Config file?

I don't want to use a constant because to test it I use the same code and pass different tunings with the env var.

Reply:
usually command line options have worked well although they can be obnoxious to plumb through everywhere.  They are easy to provide tests for, though ;)